fix: include default export in the "exports" fields

Westbrook opened the pull request:
## Description
Prep for usages in `webpack@5` and correct distribution via jspm.dev
- Add `exports` listings for default exports.
- Correct extension usage for `lit-html` directives

## Related Issue
fixes #932
refs #878 
 
## Motivation and Context
Our packages should work in all bundlers out of the box and be available via CDNs.

## How Has This Been Tested?
Manually created a `webpack@5` project locally and installed `@spectrum-web-components/bundle@0.15.4-beta.17` to test. Also tracked https://jspm.dev/npm:@spectrum-web-components/base@0.1.2-alpha.17 to actual file as opposed to what you get from https://jspm.dev/npm:@spectrum-web-components/base@0.1.1

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

adixon-adobe reviewed (commented):
I'm not totally grokking why the `alpha` version changes are necessary. The code changes look good otherwise. My one other question is what's triggering the `gitHead` changes in those package.json files.

Westbrook commented:
This is why we code review!! Those alphas are dirty commits 🙈. Will fix.

adixon-adobe commented:
Yeah -- I wasn't sure since you mentioned the alpha bits in your PR description. Glad I asked!

adixon-adobe reviewed (dismissed):
Outside of the PR checks failing, this looks good!

Westbrook commented:
Checks will hopefully clear this time 🤞 

Our docker build made a file location change https://github.com/microsoft/playwright/pull/3899/files that I'll have to prepare for later. For now, I've moved that dependency back to the "latest" release instead of the "next" release. We had moved to that in order to acquire changes they had made specifically for us, those changes have since been released publicly.
